src/components/Panel/PanelContainer.js

```diff
@@ -142,18 +152,50 @@ export default class PanelContainer extends Component {
       secondaryButton,
       subtitle,
       title,
+      hasScrollingContent,
+      panelAriaLabel,
+
+      // Explicitly define so they aren't in `...other`.
+      disableEscape,
+      labels,
+      rootNode,
+
+      // Required because it may include `aria-label`, etc.
+      ...other
     } = this.props;
 
     const hasFooter = renderFooter || primaryButton;
 
+    const ariaLabel =
+      title || this.props['aria-label'] || panelAriaLabel || subtitle;
```

Comment:
I'm curious to hear your thoughts about the differences between the `aria-label` and `panelAriaLabel` props? It initially feels like we should go with one or the other.

Also, perhaps the ordering here should be more like the following, so if a consumer supplies an `aria-label` that takes ultimate preference over everything else:

```js
this.props['aria-label'] || title || subtitle;
```

Reply:
I strongly thing we should keep the same logic as Carbon's `Modal`, to create a similar expectation with how the `Panel` is meant to be used. Their accessibility team made this decision here -- https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Modal/Modal.js#L362

```
    const ariaLabel =
      modalLabel || this.props['aria-label'] || modalAriaLabel || modalHeading;
```

(where `modalLabel` is virtually the same as the `title` here)

Reply:
As for the difference between `aria-label` and `panelAriaLabel` -- my understanding is that Carbon was trying to catch a wide net with the `Modal`'s a11y updates, providing as much flexibility as possible for folks. `aria-label` is a valid attribute (that would be used here) but it's possible that folks would prefer to use React props.

Reply:
@SimonFinney Actually disregard my last comment about `panelAriaLabel` -- I added it because I saw it used in the `Modal`, but it appears based on the git history that their equivalent, `modalAriaLabel`, is 2 years old so they are keeping it around for backwards compatibility: https://github.com/carbon-design-system/carbon/blame/4a0c095f6086c78f9bc4d2b1d7a92d89cb79d990/packages/react/src/components/Modal/Modal.js#L69

Likewise I'll go ahead and remove `panelAriaLabel` 👍

Reply:
Thanks for clarifying! I trust Carbon's decision considering they have a dedicated a11y resource, but I worry that if an `aria-label` is supplied intentionally then it's always going to be superseded by the `title`, which may not be the intention.

For example, the `aria-label` could be declared something like  'IP risk panel', while the `title` could be an IP address like '127.0.0.1', which I feel doesn't provide as much semantic context.

But... perhaps I'm over-thinking this! 😆I'd love to hear your thoughts!

Reply:
I think in this case, `aria-label` is meant to be used if you don't have a `title`. Which some folks don't have, for designs or whatever the case may be.

Reply:
Ah, that makes sense - Thanks for clarifying!